docker-compose.yml

```diff
@@ -12,6 +14,8 @@ services:
 
   rabbitmq:
     image: rabbitmq:management
+    volumes:
+      - rabbitmq-data:/var/lib/rabbitmq/mnesia
```

Comment:
I would use `.medtagger/rabbitmq-data` and please add `.medtagger` to .gitignore.